docs(network): Added a note about APN parameter in cellular doc (#5352)
eclipse/kura: docs/gateway-configuration/cellular-configuration.md

    - This port may not be used to send AT commands to the modem.
    - The _PositionService_ should be enabled. Serial settings of the _PositionService_ should not be changed; it will be redirected to the modem GPS port automatically.

### Notes about the APN parameter

> The Access Point Name is an optional parameter. If left empty, the value is automatically picked up from the Mobile Broadband Provider the modem is registered to. If a value is filled, the APN value is esplicitely configured and the default value is overridden.

Reviewer: ```suggestion
The Access Point Name is an optional parameter. If left empty, the value is automatically picked up from the Mobile Broadband Provider the modem is registered to. If a value is filled, the APN value is explicitly configured and the default value is overridden.
```

> ### Notes about the APN parameter
> The Access Point Name is an optional parameter. If left empty, the value is automatically picked up from the Mobile Broadband Provider the modem is registered to. If a value is filled, the APN value is esplicitely configured and the default value is overridden.
> 
> A good practice is to set the interface status to **Disabled** and then **Enable For WAN** when the APN is explicitely set. NetworkManager, indeed, will fallback to the default value if a wrong APN is specified, causing misleading behaviors. This does not happen if the interface is disabled and re-enabled after APN changes.

Reviewer: Would adding some styling help the user notice this note better?

```suggestion
!!! note  Notes about the APN parameter

    The Access Point Name is an optional parameter. If left empty, the value is automatically picked up from the 
    Mobile Broadband Provider the modem is registered to. If a value is filled, the APN value is explicitly 
    configured and the default value is overridden.

    A good practice is to set the interface status to **Disabled** and then **Enable For WAN** when the APN is 
    explicitly set. NetworkManager, indeed, will fallback to the default value if a wrong APN is specified, causing 
    misleading behaviors. This does not happen if the interface is disabled and re-enabled after APN changes.
```

> A good practice is to set the interface status to **Disabled** and then **Enable For WAN** when the APN is explicitely set. NetworkManager, indeed, will fallback to the default value if a wrong APN is specified, causing misleading behaviors. This does not happen if the interface is disabled and re-enabled after APN changes.

Reviewer: ```suggestion
A good practice is to set the interface status to **Disabled** and then **Enable For WAN** when the APN is explicitly set. NetworkManager, indeed, will fallback to the default value if a wrong APN is specified, causing misleading behaviors. This does not happen if the interface is disabled and re-enabled after APN changes.
```
